core/src/components/GlobalSearch/SearchResult.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+	<NcListItem class="result-items__item"
+		:name="title ?? ''"
+		:bold="false"
+		@click="openResult(result)">
+		<template #icon>
+			<div class="result-items__item-icon"
```

Comment:
Either this or set `alt=""` on the `<img>` tag. But I think hiding it is correct here as this is pure representation (images).

```suggestion
			<div aria-hidden="true"
				class="result-items__item-icon"
```